test/extended/machines/cluster.go

```diff
@@ -12,16 +12,17 @@ import (
 	"k8s.io/client-go/dynamic"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-
-	"github.com/openshift/origin/test/extended/util/ibmcloud"
 )
 
 var _ = g.Describe("[sig-cluster-lifecycle][Feature:Machines][Early] Managed cluster should", func() {
 	defer g.GinkgoRecover()
 
 	g.It("have same number of Machines and Nodes", func() {
-		if e2e.TestContext.Provider == ibmcloud.ProviderName {
-			e2eskipper.Skipf("IBM Cloud clusters do not contain machineset resources")
+		if e2e.TestContext.Provider != "aws" &&
```

Comment:
As I said in a comment in the previous PR, we need to support aws/gcp/etc UPI as well, and I am pretty sure those also don't have machine objects.

So the test should be "is machineAPI enabled"...OK I wrote up https://github.com/openshift/origin/pull/25428